test/test-setup.sh

```diff
@@ -25,7 +25,7 @@ echo
 echo " Setting up test environment..."
 
 # Build and start docker container
-sh -c "docker run -d -e SETUPSH_SHOW_DEBUG -e COMPOSE_HTTP_TIMEOUT -e DOCKER_CLIENT_TIMEOUT -e CONNECT_TO_KUZZLE_WAIT_TIME_BETWEEN_RETRY --privileged --rm --name $CONTAINER_NAME -v $PWD:/opt $IMAGE_NAME $OUTPUT"
+sh -c "docker run -d -e SETUPSH_SHOW_DEBUG -e COMPOSE_HTTP_TIMEOUT -e DOCKER_CLIENT_TIMEOUT -e CONNECT_TO_KUZZLE_MAX_RETRY --privileged --rm --name $CONTAINER_NAME -v $PWD:/opt $IMAGE_NAME $OUTPUT"
```

Comment:
nitpicking: why use a sh -c here and not directly call the `docker` command?

Reply:
🤔 I guess you're right. There's no use for the `sh -c` here, and I don't really remember why it's so. This part has been done with so much try and error... I'll take a look at it and get back.